### PR TITLE
Update de la page d'intro - terminologie

### PR DESCRIPTION
**Pull request by Cwin-mkg**
vu avec P. Laumay

### CODE REVIEW

**Comment by florentsarat** [comment]
@Cwin-mkg and @philippebonita 
We have currently two intro pages: one in the getting started (https://documentation.bonitasoft.com/bonita/7.12/what-is-bonita)and one in the Application and Process Design menu (the one you've updated - it also contains a getting started)

Can't we merge them ? And maybe put it a more highlighted part of the documentation (maybe just before the getting started -the first thing you want to know is what's Bonita)

**Comment by Cwin-mkg** [comment]
We could do this and have an intro in a separate page.



<https://bonitasoft.com>
<http://bonitasoft.com/>

Carole WINQWIST
<https://data.sigilium.com/signatures/rcRYysoJtBDkSpV5-XA8Rb829peW0rc/link?element=name>

VP Marketing
<https://data.sigilium.com/signatures/rcRYysoJtBDkSpV5-XA8Rb829peW0rc/link?element=job>

+33 6 23 46 52 05
<https://data.sigilium.com/signatures/rcRYysoJtBDkSpV5-XA8Rb829peW0rc/link?element=mobile>
carole.winqwist@bonitasoft.com
76, boulevard de la République
92100 Boulogne Billancourt
<https://data.sigilium.com/signatures/rcRYysoJtBDkSpV5-XA8Rb829peW0rc/link?element=map_marker>
<https://www.facebook.com/bonitasoftbpm>
<https://twitter.com/bonitasoft>
<https://www.youtube.com/user/BonitaSoft>

<https://data.sigilium.com/signatures/rcRYysoJtBDkSpV5-XA8Rb829peW0rc/redirect_link>

**Comment by educhastenier** [comment]
Hi @Cwin-mkg Can you make sure all you write is in English, please?
This repository is public and all its content (even Pull Requests) should be written in English.
Thx for your understanding.

**Comment by florentsarat** [comment]
Merged the two introductions/overviews + moved the resulting page up in taxonomy and added a new welcome page for the getting started

File: md/what-is-bonita.md [hunk 1]
```diff
@@ -1,59 +1,62 @@
 # What is Bonita?
```
Comment (NathalieC) [inline md/what-is-bonita.md:1]: Review done. Some changes made.

File: md/tutorial-overview.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Tutorial overview
```
Comment (NathalieC) [inline md/tutorial-overview.md:1]: Review done. Some changes made.

File: md/taxonomy.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
+* [What is Bonita?](what-is-bonita.md)
```
Comment (NathalieC) [inline md/taxonomy.md:1]: Review done, some changes made

File: md/bonita-bpm-overview.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
 # Bonita platform overview
```
Comment (NathalieC) [inline md/bonita-bpm-overview.md:1]: Now we need to remove this page from the repository. How can we make sure no links navigate to this page?

**Comment by NathalieC** [comment]
@Cwin-mkg Can you let us know if the new changes suit you?

**Review by Cwin-mkg — COMMENTED** [review]
ok for me

File: md/taxonomy.md (hunk 3 above)
Comment (Cwin-mkg) [reply md/taxonomy.md:1]: ok for me

**Review by Cwin-mkg — COMMENTED** [review]
OK for me

**Pull request merged**